www/src/Gallery.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { Form } from '@edx/paragon';
+
+import { EditorPage } from '@edx/frontend-lib-content-components';
+
+export const EditorGallery = () => {
+  const [blockType, setBlockType] = React.useState('html');
+  const blockIds = {
```

Comment:
This 1. should be imported from the file where we specify the block types. 2 should be call `blockTypes` not `blockIds`